### PR TITLE
R1 driver preferences

### PR DESCRIPTION
**Pull request by ky28059**
@pinkbluesky

### CODE REVIEW

**Comment by ky28059** [comment]
Will need to test ramp rate to dampen braking effect as opposed to current idle mode solution.

File: src/main/java/frc/robot/Constants.java [hunk 1]
```diff
@@ -23,7 +23,7 @@
  * wherever the constants are needed, to reduce verbosity.
  */
 public final class Constants {
-    public static final boolean IS_R1 = false;
+    public static final boolean IS_R1 = true;
```
Comment (e3l) [inline src/main/java/frc/robot/Constants.java:26]: let's leave this as false on main for now?

File: src/main/java/frc/robot/RobotContainer.java [hunk 2]
```diff
@@ -93,11 +93,16 @@ public class RobotContainer {
      * The container for the robot. Contains subsystems, OI devices, and commands.
      */
     public RobotContainer() {
-        driveController = new XboxDriveController();
+
+        CameraServer.startAutomaticCapture();
```
Comment (e3l) [inline src/main/java/frc/robot/RobotContainer.java:97]: can be removed now that switchable cameras is up

File: src/main/java/frc/robot/RobotContainer.java [hunk 3]
```diff
@@ -93,11 +93,16 @@ public class RobotContainer {
...
+
+        // driveController = new XboxDriveController();
+        driveController = new DualJoystickDriveController();
 
         photonWrapper = new PhotonWrapper();
 
-        driveSubsystem = new SwerveSubsystem(photonWrapper);
+        // driveSubsystem = new SwerveSubsystem(photonWrapper);
+        driveSubsystem = new MissileShellSwerveSubsystem();
```
Comment (e3l) [inline src/main/java/frc/robot/RobotContainer.java:105]: dangerous

File: src/main/java/frc/robot/RobotContainer.java [hunk 4]
```diff
@@ -141,7 +146,7 @@ public RobotContainer() {
      * passing it to a {@link edu.wpi.first.wpilibj2.command.button.JoystickButton}.
      */
     private void configureButtonBindings() {
-        driveController.getBalancerButton().whileTrue(balancerCommand);
+        // driveController.getBalancerButton().whileTrue(balancerCommand);
```
Comment (e3l) [inline src/main/java/frc/robot/RobotContainer.java:149]: dangerous

**Comment by ky28059** [comment]
I'm going to wait for #63 before rebasing this as well, because this PR swaps buttons around as well.

File: src/main/java/frc/robot/controllers/DualJoystickDriveController.java [hunk 5]
```diff
@@ -37,17 +37,19 @@ public class DualJoystickDriveController extends BaseDriveController {
...
     public double getForwardPower() {
-        return MathUtil.applyDeadband(-leftJoystick.getY(), JOYSTICK_DEADBAND);
+        double scale = leftJoystick.getTrigger() ? 1.0 : 0.75;
+        return MathUtil.applyDeadband(-leftJoystick.getY() * scale, JOYSTICK_DEADBAND);
     }
 
     @Override
     public double getLeftPower() {
-        return MathUtil.applyDeadband(-leftJoystick.getX(), JOYSTICK_DEADBAND);
+        double scale = leftJoystick.getTrigger() ? 1.0 : 0.75;
```
Comment (e3l) [inline src/main/java/frc/robot/controllers/DualJoystickDriveController.java:46]: nitpicky: maybe could be abstracted into an isSlowMode()

File: src/main/java/frc/robot/controllers/DualJoystickDriveController.java [hunk 6]
```diff
@@ -57,6 +59,7 @@ public boolean getSwerveRelative() {
 
     @Override
     public JoystickButton getBalancerButton() {
+        // TODO: this button is already bound; merge #63 and change this button in rebase
```
Comment (e3l) [inline src/main/java/frc/robot/controllers/DualJoystickDriveController.java:62]: todo?

**Review by e3l — APPROVED** [review]

File: src/main/java/frc/robot/subsystems/RollerSubsystem.java [hunk 7]
```diff
@@ -64,11 +65,12 @@ public enum HeldPiece {
 
     public RollerSubsystem() {
         leftBeak = MotorUtil.createTalonSRX(LEFT_ID);
-        leftBeak.setInverted(true);
+        leftBeak.setInverted(false);
```
Comment (ky28059) [inline src/main/java/frc/robot/subsystems/RollerSubsystem.java:68]: Is this only an R1 thing? For now I'll comment it out.

**Pull request merged**